.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -18,31 +18,44 @@ For OEM releases, keep the OEM Release template and remove the Open Release one
 
  - [ ] [DOC] Ping in #documentation-internal about the new release
  - [ ] [GIT] Create branch `release/M.m.p` in owncloud/android from master
+ - [ ] [GIT] Rebase `release/M.m.p` against `stable` in owncloud/android
+ - [ ] [GIT] Create branch `release/x.y.z` in owncloud/android-library from master
+ - [ ] [GIT] Rebase `release/x.y.z` against `stable` in owncloud/android-library
  - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
  - [ ] [DEV] Update [SBOM](https://cloud.owncloud.com/f/6072870)
  - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder
  - [ ] [DIS] Move all changelog files from the unreleased folder to the new version folder
  - [ ] [DIS] Update screenshots, if needed, in README.md
  - [ ] [DIS] Add ReleaseNotes replacing `emptyList` with `listOf` and adding inside `ReleaseNote()` with String resources
+ - [ ] [DIS] Prepare post in central.owncloud.org ([`Category:News + Tag:android`](https://central.owncloud.org/tags/c/news/5/android))
  - [ ] [QA] Design Test plan
  - [ ] [QA] Regression Test execution
- - [ ] [GIT] Create and sign tag `oc-android-M.m.p` in HEAD commit of release branch, in owncloud/android
- - [ ] [GIT] Create and sign tag `x.y.z` in HEAD commit of release branch, in owncloud/android-library
  - [ ] [DIS] Generate final bundle from signed commit in owncloud/android
+ - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
+ - [ ] [DEV] Code Review
+ - [ ] [DOC] Ping in #documentation-internal that we are close to sign the new tags
+ - [ ] [GIT] Create and sign tag `vM.m.p` in HEAD commit of release branch, in owncloud/android
+ - [ ] [GIT] Create and sign tag `vx.y.z` in HEAD commit of release branch, in owncloud/android-library
  - [ ] [DIS] Upload & publish release bundle and changelog in Play Store
  - [ ] [DIS] Update screenshots and store listing, if needed, in Play Store
  - [ ] [GIT] Publish a new release in [owncloud/android](https://github.com/owncloud/android/releases)
- - [ ] [DIS] Create post in central.owncloud.org ([`Category:News + Tag:android`](https://central.owncloud.org/tags/c/news/5/android))
- - [ ] [COM] Inform `#updates` and `#marketing` in internal chat
- - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
+ - [ ] [DIS] Release published in Google Play
```

Comment:
```suggestion
 - [ ] [DIS] Release published in Play Store
```
Let's call it always the same way to avoid confusions :D

Reply:
Sure 👍